src/SuperResponsiveTable.js

```diff
@@ -71,19 +118,50 @@ export const Tr = props => (
   <Consumer>{headers => <TrInner {...props} headers={headers} />}</Consumer>
 )
 
-export const Th = props => <th data-testid="th" {...allowed(props)} />
-export const Tbody = props => <tbody data-testid="tbody" {...allowed(props)} />
+export const Th = props => (
+  <th
+    className={classes({ bem: 'th', others: props.className })}
+    data-testid="th"
+    {...allowed(props)}
+  />
+)
+export const Tbody = props => (
+  <tbody
+    className={classes({ bem: 'tbody', others: props.className })}
+    data-testid="tbody"
+    {...allowed(props)}
+  />
+)
 
 class TdInner extends React.Component {
   render() {
     if (this.props.colSpan) {
-      return <td data-testid="td" {...allowed(this.props)} />
+      return (
+        <td
+          className={classes({
+            bem: 'td',
+            others: this.props.className,
+          })}
+          data-testid="td"
+          {...allowed(this.props)}
+        />
+      )
     }
     const { headers, children, columnKey } = this.props
-    const classes = (this.props.className || '') + ' pivoted'
+
     return (
-      <td data-testid="td" {...allowed(this.props)} className={classes}>
-        <div data-testid="td-before" className="tdBefore">
+      <td
+        className={classes({
+          bem: ['td', 'td_pivoted'],
+          others: ['pivoted', this.props.className],
+        })}
+        data-testid="td"
+        {...allowed(this.props)}
+      >
+        <div
+          className={classes({ bem: 'th_pseudo', others: 'tdBefore' })}
```

Comment:
I'm assigning this the name `th_pseudo` over the old name because I think it's more in line with the purpose it serves as the "pseudo table header".

The name `super-responsive-table__th_pseudo` will also be used by users to style these elements, so it might also be more intuitive for them.

Reply:
Or if it's not intuitive enough, maybe we can have `withBaseStyles` accept an option to set a class name for this element? 🤔 That way, we also won't have to expose the BEM naming to users.

Reply:
I think `th_term` would make sense here. "term" is inspired from the  `<dt>` [description term](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dt) element in a description list. In a future PR, I'd really like to refactor this layout to a description list. 

How does this classname sound to you?

Reply:
> In a future PR, I'd really like to refactor this layout to a description list.

I quite like this idea! I thought at first about using the [`columnheader` table role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Table_Role) to keep everything semantic, but I think converting everything to a description list is a better idea.

Though if we're leaning into exposing this class name, do you think we should also document the whole set of BEM class names then?

(I'll also add a `thTermClassName` option in the `withBaseStyles` prop so anyone using `styled-components` also has the option of doing [something like this](https://github.com/reactjs/react-modal/issues/603#issuecomment-378847885).)

Reply:
I wouldn't add the additional classname option in this PR. It's best to limit options at this time. Do document the relevant classnames for easy targeting.

I'm rethinking the use of "term". We already use a "pivoted" class in the app. Call the `th_pseudo` class `super-responsive-table__th_pivoted`. It's clear, with or without the use of description list elements. I don't want people to have to think much about non-table elements.

Reply:
> Do document the relevant classnames for easy targeting.

Done! https://github.com/ua-oira/react-super-responsive-table/pull/132/commits/08bf8454178f4b98c146727cbea3e887c14efe5c

> I don't want people to have to think much about non-table elements.

Felt the same after giving it some more thought. +1 for `pivoted` since we already used it in the `td` element! Made the changes here: https://github.com/ua-oira/react-super-responsive-table/pull/132/commits/4c13ad6d190689e3cd51123fdd98648ba2af120b.